client/components/tinymce/plugins/mentions/mentions.jsx

```diff
@@ -0,0 +1,142 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDomServer from 'react-dom/server';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import SuggestionList from './suggestion-list';
+import EditorMention from './editor-mention';
+import QueryUsersSuggestions from 'components/data/query-users-suggestions';
+import { getUserSuggestions } from 'state/users/suggestions/selectors';
+
+const Mentions = React.createClass( {
+	displayName: 'Mentions',
+
+	propTypes: {
+		siteId: React.PropTypes.number,
+		editor: React.PropTypes.object,
+		suggestions: React.PropTypes.array
+	},
+
+	getDefaultProps() {
+		return {
+			suggestions: []
+		};
+	},
+
+	getInitialState() {
+		return {
+			query: '',
+			showPopover: false
+		};
+	},
+
+	componentDidMount: function() {
+		const { editor } = this.props;
+
+		editor.on( 'keyup', this.onKeyUp );
+	},
+
+	getCurrentText: function() {
+		const rng = this.props.editor.selection.getRng( 1 );
+
+		return rng.startContainer.textContent;
```

Comment:
Only if they can not appear there. Could `getRng()` return `null` or `undefined`? What if the content is `''` (empty string).

If those are good we should be fine but being confident that things will always exist makes me nervous.